code/modules/food_and_drink/cakes.dm

```diff
@@ -1,3 +1,7 @@
+#define CAKE_MODE_CAKE 1
+#define CAKE_MODE_SLICE 2
+#define CAKE_MODE_STACK 1
```

Comment:
unused?

Reply:
these are used for the cake utility procs. it was suggested by pali to make the code a little more readable 🌸

Reply:
yes, but STACK is unused.

Reply:
it's used on lines 120 and 259 ❤︎